src/components/SignUp/SignUpForm.tsx

```diff
@@ -95,20 +96,20 @@ const SignUpForm = (): React.JSX.Element => {
 				<h2 className='center'>Welcome! Let's set up your account.</h2>
 			</div>
 			<div className='input-block'>
-				<label className='block' htmlFor='name-input'>Name</label>
-				<input
-					id='name-input'
-					type='text'
+				<TextInputComponent
+					label='Name'
 					name='name'
+					id='name-input'
 					placeholder='Your name'
-					minLength={1}
 					required
-					aria-describedby='name-input-helper-text'
+					helperMessageSlot={(
+						<div id='name-input-helper-text'>
+							<span>Please enter a valid name that is at least 1 character long.</span>
+						</div>
+					)}
```

Comment:
For the helper message, we have a [HelperMessageComponent](https://github.com/torontojs/communityhub/blob/dev/src/pages/usage-helper-message/sample-helper-message.tsx) that can be passed to the text component like so
```suggestion
					helperMessageSlot={<HelperMessageComponent variant='error' labelText='Please enter a valid name that is at least 1 character long.' />}
```

Reply:
For the css, we can target the helper component using this approach

In the css file, we can modify the lines 110, 111 as such
From 
```
#name-input-helper-text { display: none; }
#name-input:not(:focus):user-invalid ~ #name-input-helper-text { display: block; }
```

To:
```
#name-input ~ .helper-container { display: none; }
#name-input:user-invalid:not(:focus) ~ .helper-container { display: flex; }
```

Reply:
What this does is, instead of directly taking the helper container id, we target the `id` of the `text input ` and selects any sibling elements with the class `helper-container` with respect to the text input.

Reply:
Alternatively, we can modify the HelperMessageComponent to include an `id` param and then use it to target the element directly.  In this case we won't need to modify the existing css.

Which would be a more precise approach. I would recommend this one over the other.

Reply:
@roshan-budhathoki This was not a part of the task itself. I will let you decided if you want to handle it here or create a separate issue for it.

Reply:
I will handle it here @manvinderjit. It would be easier for me.

Reply:
> Alternatively, we can modify the HelperMessageComponent to include an `id` param and then use it to target the element directly. In this case we won't need to modify the existing css.

"Helper Message Component" is used on multiple places. So, I would not touch it at the moment but will keep your thought in consideration.

Reply:
> > Alternatively, we can modify the HelperMessageComponent to include an `id` param and then use it to target the element directly. In this case we won't need to modify the existing css.
> 
> "Helper Message Component" is used on multiple places. So, I would not touch it at the moment but will keep your thought in consideration.

@roshan-budhathoki  Please create a GitHub issue for this.